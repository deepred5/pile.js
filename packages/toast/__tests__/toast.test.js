import { shallow } from 'enzyme';
import React from 'react';
import ReactDOM from 'react-dom';
import ToastContent from '../src/toasContent';

describe('test toast component', () => {
  beforeAll(() => {
    ReactDOM.createPortal = jest.fn(element => element);
  });

  afterEach(() => {
    ReactDOM.createPortal.mockClear();
  });
  it('snapshot toast ui', () => {
    const wrapper = shallow(<ToastContent content="hi,you" iconType="success" />);
    expect(wrapper).toMatchSnapshot();
  });
  it('', () => {

  });
});
