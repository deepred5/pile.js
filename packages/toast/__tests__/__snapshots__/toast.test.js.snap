// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test toast component snapshot toast ui 1`] = `
<div
  className="pile-toast"
>
  <Icon
    prefixCls="pile"
    size="large"
    type="check-circle"
  />
  <div
    className="pile-toast-content"
  >
    hi,you
  </div>
</div>
`;
